<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Fem_VisualizeClass</class>
 <widget class="QWidget" name="Fem_VisualizeClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>957</width>
    <height>807</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Fem_Visualize</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,4">
   <item>
    <widget class="QWidget" name="widget_2" native="true">
     <widget class="QPushButton" name="pushButtonDis">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>161</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>变形</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonShow">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>161</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>消隐</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonForce">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>130</y>
        <width>161</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>轴力</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonRea">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>190</y>
        <width>161</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>反力</string>
      </property>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QVTKOpenGLNativeWidget" name="widget" native="true"/>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QVTKOpenGLNativeWidget</class>
   <extends>QWidget</extends>
   <header>qvtkopenglnativewidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="Fem_Visualize.qrc"/>
 </resources>
 <connections/>
</ui>
